dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -58,7 +58,7 @@ def create_glove_char(n_dims: int, source_file: str = None) -> None:
     embd_table = build_embd_dictionary(source_file)
     embd_words: List[str]
     embd_matrix: List[np.ndarray]
-    embd_words, embd_matrix = [
+    embd_words, embd_matrix = [  # type: ignore
```

Comment:
don't love the `#type: ignore`. If there isn't a way to avoid this, that's okay. But IIRC, in prior PRs we are tried to avoid this, @Sanketh7?

Reply:
The issue I was having was that there's just a lot of types involved and numpy's type annotations aren't very specific to help mypy out. Mypy sort of gives up and says that you're trying to assign an `object` to `List[str]` and `List[np.ndarray]`.

Reply:
i see